lmdeploy/cli/entrypoint.py

```diff
@@ -16,6 +16,12 @@ def run():
     parser = CLI.parser
     args = parser.parse_args()
 
+    if hasattr(args, 'model_name'):
```

Comment:
Can we modify codes inside? APIs may suffer the same problem.

Reply:
pipeline api doesn't need model_name. `model_name` is only used by `v1/models`
If we launch api_server through `python -m lmdeploy.serve.openai.api_server internlm/internlm2_5-1_8b-chat`, the result  of `v1/models` is what we expect "internlm/internlm2_5-1_8-chat". 

But if we launch the server by `lmdeploy serve api_server`, `v1/models` returns the full local path of the model, because it always try to `get_model` if the `args.model_path` doesn't exist
```
if model_path is not None and not os.path.exists(args.model_path):
            args.model_path = get_model(args.model_path,
                                        download_dir=download_dir,
                                        revision=revision)
        model_path_or_server = getattr(args, 'model_path_or_server', None)
````